app/services/EmailVerificationService.scala

```diff
@@ -64,18 +66,27 @@ class EmailVerificationService @Inject()(
     validateEmailConnector.getStatus(credId)
   }
 
-  def isEmailVerified(emailAddress: String, credId: String)(implicit hc: HeaderCarrier): Future[Boolean] = {
+  def isEmailVerified(emailAddress: String, credId: String)(implicit hc: HeaderCarrier): Future[PasscodeAttemptsStatus] = {
     getStatus(credId).map {
       case Right(Some(verificationStatus)) =>
-        verificationStatus.emails.exists {
-          case emailStatus@EmailStatus(_, true, _) if emailStatus.emailAddress == emailAddress => true
-          case _ => false
+        (verificationStatus.emails.count(_.locked) >= Constants.emailVerificationMaxEmails,
+          verificationStatus.emails.exists(_.locked),
+          verificationStatus.emails.exists(_.verified)) match {
+          case (true, true, false) =>
+            PasscodeAttemptsStatus.LockedTooManyLockedEmails
```

Comment:
can we add some info logging on these various status', and can we add EmailVerificationService and set to INFO within 
 application-json-logger.xml think it will be beneficial just to see the early users complete through the process